packages/testlab/test/testlab.smoke.js

```diff
@@ -1,8 +1,8 @@
-const should = require('should');
-const testlab = require('..');
+const assert = require('assert');
 
 describe('testlab smoke test', () => {
   it('exports expect interface', () => {
-    testlab.expect.should.be.type('function');
+    const testlab = require('..');
+    assert.equal(typeof testlab.expect, 'function');
   });
 });
```

Comment:
> Ideally, I want to export do this in index.js:
```js
const testlab = module.exports = require('./lib');
testlab.es2015 = testlab.es6 = require('./lib-es2015');
```

That does not make any sense to me. Here is what I would expect myself:

```js
const nodeMajorVersion = +process.versions.node.split('.')[0];
module.exports = nodeMajorVersion >= 7 ?
  require('./lib/testlab') :
  require('./lib-es2015/testlab');
```

The proposed code works perfectly fine for me, both on Node 6 and 7.

Remember, different transpilation targets (`lib` vs `lib-es2015`) must be transparent to users of our modules. All they need is `require('@loopback/testlab')`, it's the responsibility of the loaded module to decide which code to export depending on Node version.